src/patternfly/components/Progress/progress.scss

```
@@ -140,6 +140,16 @@
     --#{$progress}__indicator--BackgroundColor: var(--#{$progress}--m-danger__indicator--BackgroundColor);
     --#{$progress}__status-icon--Color: var(--#{$progress}--m-danger__status-icon--Color);
     --#{$progress}--m-inside__measure--Color: var(--#{$progress}--m-danger--m-inside__measure--Color);
+
+    .#{$progress}__bar {
+      @media (prefers-reduced-motion: no-preference) {
+        @include pf-v6-animate-danger-jiggle;
+      }
+
+      .pf-m-status {
```

Same comment here about styling a block/element class along with the modifier so we don't accidentally target elements we don't intend to.

---

Looks like this is meant to target the `__status-icon` element

```suggestion
      .#{$progress}__status-icon {
```